Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/install_hooks.py

```diff
@@ -201,13 +207,13 @@ def main(cls, args, repository, hooks=None, identifier_template=None, **kwargs):
                     source_remotes=source_remotes,
                 )
 
-            target = os.path.join(repository.common_directory, 'hooks', hook)
             if not os.path.exists(os.path.dirname(target)):
                 os.makedirs(os.path.dirname(target))
             with open(target, 'w') as f:
                 f.write(contents)
                 f.write('\n')
             os.chmod(target, 0o775)
+            installed_hooks += 1
 
-        print('Successfully installed {} repository hooks'.format(len(hook_names)))
+        print('Successfully installed {} of {} repository hooks'.format(installed_hooks, len(hook_names)))
```

Comment:
`Successfully installed 8 of 10 repository hooks` might worth mentioning if I can proceed or this some sort of failure.

Reply:
Good point, will add a line mentioning that this is an expected condition.